frontend/src/components/editor-page/sidebar/users-online-sidebar-menu/users-online-sidebar-menu.tsx

```diff
@@ -33,28 +34,35 @@ export const UsersOnlineSidebarMenu: React.FC<SpecificSidebarMenuProps> = ({
   const buttonRef = useRef<HTMLButtonElement>(null)
   const realtimeUsers = useApplicationState((state) => state.realtimeStatus.onlineUsers)
   useTranslation()
+  const ownUsername = useApplicationState((state) => Optional.ofNullable(state.user?.username).orElseGet(() => ''))
+  const ownDisplayname = useApplicationState((state) => state.realtimeStatus.ownUser.displayName)
+  const ownStyleIndex = useApplicationState((state) => state.realtimeStatus.ownUser.styleIndex)
 
   useEffect(() => {
-    buttonRef.current?.style.setProperty('--users-online', `"${realtimeUsers.length}"`)
+    buttonRef.current?.style.setProperty('--users-online', `"${realtimeUsers.length + 1}"`)
   }, [realtimeUsers])
 
   const hide = selectedMenuId !== DocumentSidebarMenuSelection.NONE && selectedMenuId !== menuId
   const expand = selectedMenuId === menuId
   const onClickHandler = useCallback(() => onClick(menuId), [menuId, onClick])
 
+  // ToDo: use real color
+  const ownUserElement = useMemo(
+    () => (
+      <SidebarButton key={ownUsername}>
+        <UserLine displayName={ownDisplayname} username={ownUsername} color={ownStyleIndex} active={true} />
+      </SidebarButton>
+    ),
+    [ownDisplayname, ownUsername, ownStyleIndex]
+  )
+
   const onlineUserElements = useMemo(() => {
     if (realtimeUsers.length === 0) {
-      return (
-        <SidebarButton>
-          <span className={'ms-3'}>
-            <Trans i18nKey={'editor.onlineStatus.noUsers'}></Trans>
-          </span>
-        </SidebarButton>
-      )
+      return <></>
     } else {
       return realtimeUsers.map((realtimeUser) => {
         return (
-          <SidebarButton key={realtimeUser.styleIndex}>
+          <SidebarButton key={realtimeUser.username}>
```

Comment:
this will break if a logged in user is online multiple times. Just use the array index instead.

Reply:
This will be the same as well, if you have more than 8 users